### PR TITLE
Put most recent 20 messages in notif

### PR DESCRIPTION
**Pull request by dbkr**
Fixes https://github.com/vector-im/vector-web/issues/1648

### CODE REVIEW

File: synapse/storage/event_push_actions.py [hunk 1]
```diff
@@ -191,7 +192,7 @@ def get_no_receipt(txn):
                 "actions": json.loads(row[3]),
                 "received_ts": row[4],
             } for row in after_read_receipt + no_read_receipt
-        ])
+        ][0:limit])
```
Comment (erikjohnston) [inline synapse/storage/event_push_actions.py]: Generally you write `[:limit]`, but no biggie

**Comment by erikjohnston** [comment]
LGTM, assuming tests pass

**Comment by dbkr** [comment]
Sorry, realised this wouldn't work: it would end up not returning the most recent 20. This should be a bit better.

**Comment by erikjohnston** [comment]
LGTM

**Pull request merged**